pages/index.py

```diff
@@ -61,6 +61,37 @@ def slider_item(title, text, active=""):
 <!--HOME SECTION START-->
 <section id="home">
 <div class="container">
+<div class="row">
+<!--DONATE SUBSECTION -->
+<div class="col-sm-12 col-md-12 col-lg-12">
+        <div class="item active donate" id="donate_button">
+             <p>$donate_text</p>
```

Comment:
Ah, I see, it's a copy-paste. Don't bother fixing it in this pull request then, it's unrelated to the goals intended with your changeset.

There are some other HTML validity errors that I'll try to look after tomorrow so I'll probably catch this one as well.